test/Unit/Task/PhpMdTest.php

```diff
@@ -27,6 +27,7 @@ public function provideConfigurableOptions(): iterable
             [
                 'whitelist_patterns' => [],
                 'exclude' => [],
+                'report_format' => 'text',
```

Comment:
Can you add a case for baseline in `provideExternalTaskRuns` as well?

The provider is added to validate that given the task config (first argument), the command line tool is being called with the correct CLI arguments (last argument).

For every config option, we require a task to make sure that the CLI is triggered correctly.

Reply:
Will do later